src/map/script.c

```diff
@@ -11372,10 +11372,30 @@ static BUILDIN(gettimestr)
 static BUILDIN(openstorage)
 {
 	struct map_session_data *sd = script->rid2sd(st);
-	if (sd == NULL)
+	int storage_id = script_getnum(st, 2);
```

Comment:
Why not making this optional, as you need to have a storage 1 configured (SQL, for example) and considering it would improve compatibility?

```suggestion
	int storage_id = script_hasdata(st, 2) ? script_getnum(st, 2) : 1;
```

Reply:
Having this as optional is a way, but it won't tell others something is wrong.
Specifying storage id is better as it's clear that storage 1 will be used.
You can have any id you want, (even remove the Id 1), so having this optional would make it more confusing